packages/contentful--app-scripts/utils/cache-credential/index.ts

```diff
@@ -1,9 +1,9 @@
-const dotenv = require('dotenv');
-const fs = require('fs');
-const { EOL } = require('os');
+import dotenv from 'dotenv';
+import fs from 'fs';
+import { EOL } from 'os';
 const ignore = require('ignore')();
```

Comment:
Any reason for not changing this import?

Reply:
missed this one.